.circleci/config.yml

```diff
@@ -1,6 +1,7 @@
 version: 2.1
 orbs:
   aws-ecr: circleci/aws-ecr@7.3.0
+  orb-tools: circleci/orb-tools@10.0.4
```

Comment:
As of June, `orb-tools` is now on [version 11.x](https://circleci.com/developer/orbs/orb/circleci/orb-tools?version=11.2.0)
It's very much a breaking change! 😱

It looks a lot simpler, with far fewer jobs but their behaviours more controlled by parameters. It may be worth a look. I think it'd be nice to be on it, always a bit frustrating to start off a step behind, but I appreciate it's more work so maybe one for another day.

I only noticed because of the change in a parameter name when I was looking something up for a different comment further down.

At the least, it may be better to get to 10.1.0 for now, though.

Reply:
Updated .circleci/config.yml to reflect comment